### PR TITLE
HW-10: final

### PR DESCRIPTION
**Pull request by rogerosha**
Hello! My project living on this site: https://rogerosha.github.io/fe-course-basic-2023/

I made it adaptive for phones and tablets, hope everything works well!

Thank you for everything you did! It's an incredible experience🌷

### CODE REVIEW

**Comment by anna-vrana-moc** [comment]
@rogerosha thank you for the good job!

File: docs/rogerosha/css/calculator.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  color: #EF4934;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  gap: 16px;
+  padding-bottom: 52px;
+}
+
+.content {
+  max-width: 1280px;
```
Comment (anna-vrana-moc) [inline docs/rogerosha/css/calculator.css:63]: Screenshot from 2024-01-08 07-34-35
Please add     `min-height: calc(100vh - 250px); `to fix the alignment of the footer

**Review by anna-vrana-moc — CHANGES_REQUESTED** [review]

**Comment by anna-vrana-moc** [comment]
Hey, teams @MZabolotnev @oleg-nikulin-moc @vladyslav-yermolin-moc @vadym-zinchenko-moc ! I have one small suggestion about footer, so I believe you can take look as well

**Comment by vladyslav-yermolin-moc** [comment]
Hey, @anna-vrana-moc, I'm afraid, the deadline for mentors deep review was yesterday. So I will merge this pr, after small change described in my comment.

File: docs/rogerosha/css/calculator.css (hunk 1 above)
Comment (rogerosha) [reply docs/rogerosha/css/calculator.css:63]: Sorry for the delay, everything is done 👌

File: homeworks/kostiantyn.nosatskyi_NosatskyiK/4-js_basics/basics.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// script for index.html
```
[the remark this answers is not in the document]
Comment (rogerosha) [reply homeworks/kostiantyn.nosatskyi_NosatskyiK/4-js_basics/basics.js]: Done! Sorry for the inattention 😇

**Review by vladyslav-yermolin-moc — APPROVED** [review]
Thanx for your work, @rogerosha, congratulations!

**Pull request merged**